prototype2/gdgem/NMXConfig.h

```diff
@@ -50,11 +51,14 @@ struct EventFilter {
 };
 
 struct NMXConfig {
-  NMXConfig() {}
-  NMXConfig(std::string jsonfile);
+  NMXConfig() { }
+  NMXConfig(std::string configfile, std::string calibrationfile);
```

Comment:
Does it really make sense to have this configuration in a separate file? Why not have the calibration in the same config file as the other stuff?

Reply:
Maybe, maybe not. I like the idea of handling calibration file loading in a separate class. Maybe mainly for unit testing and to maintain classes small. But it could also be argued that it logically belongs to NMXConfig....what to do?

Reply:
That's a minor thing. I don't mind so much that it is a separate file, only that it may come at a price of complicating the CLI and command server or whatever else by requiring NMX-specific and thus non-general behavior.